onnx/runtime/aionnx/op_sequence_insert.py

```diff
@@ -0,0 +1,43 @@
+# SPDX-License-Identifier: Apache-2.0
+# pylint: disable=W0221
+
+from typing import Any, List
+
+import numpy as np  # type: ignore
+
+from ..op_run import OpRun
+
+
+def sequence_insert_reference_implementation(
+    sequence: List[Any], tensor: np.ndarray, position: np.ndarray = None
```

Comment:
```suggestion
    sequence: Union[np.ndarray, Sequence], tensor: np.ndarray, position: np.ndarray = None
```

may be this since it can be np.ndarray?

Reply:
done